torchinductor/triton_ops/autotune.py

```diff
@@ -320,8 +321,13 @@ def reduction_heuristics(size_hints, reduction_hint=False, filename=None):
         contiguous_config = triton_config_reduction(
             size_hints, 1, (rnumel if 256 <= rnumel < 2048 else 2048), num_stages=1
         )
+        outer_config = triton_config_reduction(size_hints, 128, 8)
         if reduction_hint == ReductionHint.INNER:
             return apply_triton_config(contiguous_config)
+        elif reduction_hint == ReductionHint.OUTER:
+            return apply_triton_config(outer_config)
+        elif reduction_hint == ReductionHint.OUTER_TINY:
+            return apply_triton_config(size_hints, 2 * (256 // rnumel), rnumel)
```

Comment:
Won't `256 // rnumel` be `0` if rnumel==512?  Perhaps we should guard this based on `rnumel<=256`?

Reply:
Yeah I initially used tiny hint only for rnumel less than 256, but then tweaked it some more, I'll make sure the numbers agree.